Preciso Bid Adapter: update

PrecisoSRL opened the pull request:
<!--
Thanks for improving the documentation 😃
Please give a short description and check the matching checkboxes to help us review this as quick as possible.

Please make the PR writeable. This allows us to fix typos, grammar and linting errors ourselves, which makes
merging and reviewing a lot faster for everybody.
-->

## 🏷 Type of documentation
<!-- Remove items that don't apply and/or select an item by changing [ ] to [x] -->
- [ ] new bid adapter
- [x] update bid adapter
- [ ] new feature
- [ ] text edit only (wording, typos)
- [ ] bugfix (code examples)
- [ ] new examples

## 📋 Checklist
<!-- Remove items that don't apply and/or select an item by changing [ ] to [x] -->
- [ ] Related pull requests in prebid.js or server are linked -> Paste link in this list or reference it on the PR itself
- [ ] For new adapters check [submitting your adapter docs](https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter)

bretg commented on dev-docs/bidders/precisonat.md:
```diff
@@ -0,0 +1,124 @@
+| Name          | Scope    | Description         | Example       | Type     |
+|---------------|----------|---------------------|---------------|----------|
+| `publisherId` | required | Numeric Publisher ID <br>(as provided by Preciso)  | `'123AB123'`    | `string` |
+| `region`      | optional,recommended | 3 letter country code     | `'USA'` | `string` |
```
FWIW, your adapter md file says region is required -- [modules/precisonatBidAdapter.md ](https://github.com/prebid/Prebid.js/pull/12208/files#diff-3fce2778d802e22e6f9d004e65d50b58243bc7166e245fd4ffdbb579dc286994)

But I don't see it actually getting used within the js code. The doc here is ok, but please update your Prebid.js pull request to note that region is optional.

bretg commented on dev-docs/bidders/precisonat.md:
```diff
@@ -0,0 +1,124 @@
+
+Notes:
+
+- It is best to provide the global `bcat` and `badv` within the OpenRTB fields (below). When both methods are provided, ORTB values will be prioritized.
```
Would also like for you to note that bidFloor would best be supplied by the Prebid Floors module and currency by the Currency module.

PrecisoSRL commented:
Hi @bretg ,
Thank you for your review and valuable suggestions. I have added code to support native media types in the existing adapter instead of creating a new one. I have also modified the markdown files accordingly.

Thank you!
